### ISSUE TITLE
Updated design?

### ISSUE THREAD

**Issue by chunhaow** [polvinc/HEMY#4]
Dear Polvinc,

Thank you so much for this great design. Just wondering if you could release the updates/modifications you made to the mount and the base. I plan to CNC the parts and build one soon, and it would be great if I didn't build the outdated version. For the electronics, I can 3d print a top plate and use other solutions. Thanks a lot!

**Comment by polvinc** [comment]
Hi, there,
Thanks for your interest in HEMY. I don't have time to commit all the updates yet. One or two more weeks of patience, is that OK with you? I'm still reviewing the electronics and mechanics before publishing.
But here's what it looks like :

![HEMYv1 1_update_00001](https://github.com/user-attachments/assets/6633780d-2579-4cd3-b9d1-7144f6c20b07)
![HEMYv1 1_update_00002](https://github.com/user-attachments/assets/20c400c7-237a-4062-8858-9b6a0066646f)
![HEMYv1 1_update_00003](https://github.com/user-attachments/assets/887e1d3f-e384-40a4-af84-7d4b9fb560bd)
![HEMYv1 1_update_00004](https://github.com/user-attachments/assets/a49abfd5-5a1e-4253-b52a-87fe9cf538e7)
![HEMYv1 1_update_00005](https://github.com/user-attachments/assets/0523eec2-1d53-438c-85cd-78f845e18b31)
![HEMYv1 1_update_00006](https://github.com/user-attachments/assets/03b10d85-9924-4ba1-b451-93f9f8a81a17)
![HEMYv1 1_update_00007](https://github.com/user-attachments/assets/98fbe254-1880-4955-a563-90020f826834)
![HEMYv1 1_update_00008](https://github.com/user-attachments/assets/3bae3c1f-0849-4f90-be21-3d08c6e87405)
![HEMYv1 1_update_00009](https://github.com/user-attachments/assets/90ecbc9e-6b5f-4f6c-aa4a-f205eca76a4d)
![HEMYv1 1_update_00010](https://github.com/user-attachments/assets/e77879c2-6d2b-4c4f-a20b-cfc0a937df0f)

**Comment by chunhaow** [comment]
Thank you so much for the prompt reply! This looks great and promising! Looking forward to your commits.

**Comment by polvinc** [comment]
I've just pushed the HEMY 1.1 upgrade, if you're interested in getting started. Mind you, even though I've done a detailed review, I haven't yet received my parts, nor the electronics. I should receive everything next week. So the work is not yet totally validated!

**Comment by chunhaow** [comment]
Thanks a lot! I plan to CNC the parts soon, but have a few questions:

1. I saw the base parts have 2D drawing. I am wondering if you could also provide the 2D draws for the mount parts. Without these draws, they cannot figure out the diameters of the holes and where to tap threads, etc.

2. For the PCB, I can just build it from https://oshwlab.com/lordzurp/teenastro-redux right?

3. But what about the brake driver. Is it available somewhere?

Thanks again!

**Comment by polvinc** [comment]
You're right, I haven't finished the “clean” drawings for the mount yet. For my machining, I did it case by case. Properly detailed drawings shouldn't take too long to be pushed onto the git, I'm on it.

As for the electronics, the TeenAstro Redux project we finished with LordZurp is not yet officially published, since we haven't yet been able to really test the electronics (especially the Brake Driver part). Nevertheless, I've put the Gerbers on the git (TeenAstro Redux, Brake Driver, and Smart Hand Controller). At your own risk if you start manufacturing before our tests! I made the PCBs with JLCPCB. Since I'm receiving my batch of pcb's by this WE, we should be releasing the official project shortly.
